dotnet.sh

```diff
@@ -2,14 +2,12 @@
 
 working_tree_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-echo "Running init-tools.sh"
-source $working_tree_root/init-tools.sh
+echo "Running tools.sh"
+source $working_tree_root/eng/configure-toolset.sh
+source $working_tree_root/eng/common/tools.sh
 
-toolRuntime=$working_tree_root/Tools
-dotnet=$toolRuntime/dotnetcli/dotnet
-
-echo "Running: $dotnet $@"
-$dotnet "$@"
+echo "Running: dotnet $@"
+dotnet "$@"
```

Comment:
I think this should be using dotnet from arcade.

Reply:
That is the intention.  Arcade is supposed to put dotnet on the path.

Reply:
This wasn't working as it was missing the InitializeBuildTools call